ironfish-rust-nodejs/src/lib.rs

```diff
@@ -35,6 +40,20 @@ impl Key {
     }
 }
 
+impl ToObjectExt for mining::MineHeaderResult {
```

Comment:
Is there some way to say that Key implements this trait as well, since it matches?

Reply:
Good call, totally glossed over that. Added to Key